gulpfile.js

```diff
@@ -63,10 +63,29 @@ gulp.task('build', ['clean'], function(done){
   );
 });
 
+/**
+ * Added to pass additional custom scripts.
+ * Specifically needed for Intl polyfill https://github.com/angular/angular/issues/3333.
+ * @returns a stream of Vinyl files that can be piped.
+ */
+function scripts() {
+    return copyScripts(
+        {
+            src: [
+                'node_modules/es6-shim/es6-shim.min.js',
+                'node_modules/zone.js/dist/zone.js',
+                'node_modules/reflect-metadata/Reflect.js',
+                'node_modules/intl/dist/Intl.min.js',
+                'node_modules/intl/locale-data/jsonp/en.js'
```

Comment:
This is the right way to fix the Intl bug (https://github.com/angular/angular/issues/3333).

Previously I referenced these last 2 JS files from the index.html file, but node_modules wasn't accessible from the built page, only during testing.

Now these files get bundled up in the single app bundle JS file.